01-backup.md

```diff
@@ -54,6 +54,23 @@ we're telling Git:
 
 The four commands above only need to be run once:
 the flag `--global` tells Git to use the settings for every project on this machine.
+The configuration options get stored in your home directory in a text file called `.gitconfig`.
+
+~~~ {.bash}
+$ cat ~/.gitconfig
+~~~
+~~~ {.output}
+[user]
+	name = Vlad Dracula
+	email = vlad@tran.sylvan.ia
+[color]
+	ui = true
+[core]
+	editor = nano
+~~~
+
+This file can be edited directly.
```

Comment:
Sorry, I don't see anything in `instructors.md`. Do you mean to remove the point I added in `index.md` about how git is good for setting up parallel branches?

I thought it would be good to mention it in passing because one of Git's biggest strengths is cheap branching, but I can remove it if you want.